draft-ietf-quic-transport.md

```diff
@@ -2350,15 +2401,28 @@ Sequence:
   the value selected during the handshake comes immediately before the first
   value that a server can send.
 
+Length:
+
+: An 8-bit unsigned integer containing the length of the connection ID.  Values
+  less than 4 and greater than 18 are invalid and MUST be treated as a
+  connection error of type PROTOCOL_VIOLATION.
```

Comment:
It's a bit unfortunate that this length is a different encoding than DSIL and SSIL. Nothing to do about it I suppose.